src/main.rs

```diff
@@ -67,7 +69,18 @@ fn validate_simulation() {
 
 /// Simulate the rabbit population
 fn simulate(starting_rabbits: i128) -> i32 {
-    todo!()
+    //todo!()
+    let mut rabbits = starting_rabbits;
+    let mut days =  0;
+    while rabbits > 1 {
+    	if rabbits % 2 == 1 {
+    	    rabbits = (3 * rabbits) + 1
+    	} else {
+    	    rabbits = rabbits / 2
```

Comment:
One thing I saw from some other students (that I didn't think to do myself) was to do `rabbits /= 2;` on this line. Kind of neat!